### ISSUE TITLE
How to skip a step

### ISSUE THREAD

**Issue by LionHeart55** [PyTorchLightning/pytorch-lightning#3536]
## ❓ Questions and Help
I have a medical dataset with lots of blank targets
I would like to skip train_step(s) for these blank targets
I get the targets by splitting each batch to input, target, weight

How can I skip these batches for trainin_step or validation_step

I tried returning None instead of the regular dictionary:     {loss:  0.234, ...}

**Comment by Borda** [comment]
I would recommend doing this filtering on DataModule side, just to don't include these images in training/validation...
cc: @nateraw

**Comment by carmocca** [comment]
If you can, you should filter at the dataset, DataLoader or DataModule side. Otherwise, I have a (currently work-in-progress) PR to support this: https://github.com/PyTorchLightning/pytorch-lightning/pull/3527
